collector/kernel/bpf_src/render_bpf.h

```diff
@@ -37,8 +37,6 @@
 #define TABLE_SIZE__STACK_TRACES 16384   // Number of stack traces to keep in the table
 #define TABLE_SIZE__NIC_INFO_TABLE 128   // Info per network interface
 
```

Comment:
It looks like TABLE_SIZE__PID_INFO on line 27 is no longer used and can be removed.